### PR TITLE
1.0.2

### PR DESCRIPTION
**Pull request by wellsjo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,20 @@
...
 
-### Examples
-Websocket streaming
-- [trades firehose](examples/firehose)
-- [kraken trades](examples/kraken-trades)
-- [live orderbook](examples/orderbook)
-
-Websocket trading
-- [simple example](examples/trading)
-- [trading terminal app](examples/trading-tui)
```
Comment (dimonomid) [inline README.md]: Why the `trading-tui` was removed from readme? Imo it's one of most useful examples, I personally use it for my own stuff all the time :)

File: proto/cryptowatch/inner/resource/broker/v1beta/funding_type.proto [hunk 2]
```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
```
Comment (dimonomid) [inline proto/cryptowatch/inner/resource/broker/v1beta/funding_type.proto]: Wondering if all that `proto/cryptowatch/inner` directory needs to be made public. Does it?

File: README.md (hunk 1 above)
Comment (wellsjo) [reply README.md]: going to add it back when we release trading for real. as of now, everyone would get access denied.

File: proto/cryptowatch/inner/resource/broker/v1beta/funding_type.proto (hunk 2 above)
Comment (wellsjo) [reply proto/cryptowatch/inner/resource/broker/v1beta/funding_type.proto]: you're right, it shouldn't be public. I guess I have to modify the auto gen script for that

File: client/rest/rest.go [hunk 3]
```diff
@@ -77,6 +112,24 @@ type orderbookServer struct {
...
 }
 
+type marketSummariesServer struct {
+	Result map[string]marketSummariesItemServer `json:"result"`
+}
+
+type marketSummariesItemServer struct {
+	Price struct {
+		Last   json.Number `json:"last"`
+		High   json.Number `json:"High"`
```
Comment (wellsjo) [inline client/rest/rest.go]: I just noticed this should be `high` and `low`

Comment (wellsjo) [reply client/rest/rest.go]: done

**Pull request closed without merging**